src/api/fixtures/initial.json

```diff
@@ -614,6 +614,7 @@
     {
         "model": "api.story",
         "pk": 1,
+        "is_checkpoint": true,
```

Comment:
The object attribute should be inside "fields" object:
`"fields":{
    "is_checkpoint": true,
    "content": [...]
}`

Reply:
This way, it keeps the implemented style, in which the object attributes are located in "fields"

Reply:
OK! I'm pushing these corrections.